Add setting to indicate a flag can be set multiple times (#123)
sindresorhus/meow: readme.md

- `type`: Type of value. (Possible values: `string` `boolean`)
- `alias`: Usually used to define a short flag alias.
- `default`: Default value when the flag is not specified.

> - `multiple`: To indicate that a flag can be passed multiple times, value will be an array. Default value is `false`

Reviewer: ```suggestion
- `multiple`: Indicates that a flag can be passed multiple times. The flag value will be an array. (Default: `false`)
```
